website/docs/pact_nirvana/step_6.md

```diff
@@ -1,12 +1,34 @@
 ---
-title: 6. Platinum - Add can-i-deploy with branch tag to PR pipelines
-sidebar_label: 6. Platinum - Add can-i-deploy with branch tag to PR pipelines
+title: 6. Platinum - Add can-i-deploy with branch to PR pipelines
+sidebar_label: 6. Platinum - Add can-i-deploy with branch to PR pipelines
 ---
 
 Before we merge a PR, it would be good to know if this change is compatible with the consumers and/or providers the application talks to.
 
 We can answer this question using the can-i-deploy feature of Pact.
 
+The Pact way of managing these dependencies is to use the Pact Matrix - this is the matrix generated when you create a table of all the consumer and provider versions that have been tested against each other using Pact. You can view the Pact Matrix for any pair of applications by opening `/matrix/provider/PROVIDER/consumer/CONSUMER` in your Pact Broker.
+
+<details open>
+  <summary>Pact matrix</summary>
+
+![Pact matrix](/img/pact-matrix.png)
+</details>
+
+One very important thing to note is that a verification is associated with the pact _content_ itself, not with a specific consumer version. This means that if a pact does not change between publications, any previous verifications are automatically applied to the new pact publication, effectively “pre-verifying” it. \(For this reason, it is best not to use any random data in a pact, as this will cause the broker to consider it a new revision of the pact.\) Linking a verification to the pact content rather than to the application version also means that we can do a “cartesian join” of pacts/verifications, resulting in many more "compatible" versions than would otherwise be the case.
```

Comment:
```suggestion
### Pre-verified contracts
One very important thing to note is that a verification is associated with the pact _content_ itself, not with a specific consumer version. This means that if a pact does not change between publications, any previous verifications are automatically applied to the new pact publication, effectively “pre-verifying” it. \(For this reason, it is best not to use any random data in a pact, as this will cause the broker to consider it a new revision of the pact.\) Linking a verification to the pact content rather than to the application version also means that we can do a “cartesian join” of pacts/verifications, resulting in many more "compatible" versions than would otherwise be the case.
```